rest-api-spec/src/main/resources/rest-api-spec/api/cluster.delete_decommission_awareness.json

```diff
@@ -0,0 +1,19 @@
+{
+  "cluster.delete_decommission_awareness": {
+    "documentation": {
+      "url": "https://opensearch.org/docs/latest/opensearch/rest-api/decommission/",
```

Comment:
This link redirects to docs. Is the documentation work in progress? or Am I missing something?